src/main/java/io/confluent/connect/jdbc/source/DataConverter.java

```diff
@@ -58,7 +64,7 @@ public static Schema convertSchema(
     return builder.build();
   }
 
-  public static Struct convertRecord(Schema schema, ResultSet resultSet, boolean mapNumerics)
+  public static Struct convertRecord(Schema schema, ResultSet resultSet, NumericMapping mapNumerics)
```

Comment:
Since this PR changes the logic of numeric conversions - maybe add a couple of unit-tests, exercising conversion of different numeric literals.